src/script/components/list/ConversationListCell.tsx

```diff
@@ -167,15 +144,15 @@ const ConversationListCell: React.FC<ConversationListCellProps> = ({
     }
   };
 
+  const isFocused = currentFocusId === conversation.id;
```

Comment:
What about passing `isFocused` to this component? I don't think it needs to know what `currentFocusId` is, but rather if its focused or not.